packages/client/src/Config.ts

```diff
@@ -111,6 +111,12 @@ export interface ControlLayerConfig {
      * port mappings on the public side.
     */
     externalIp?: string
+
+    /**
+     * The maximum time to wait when forming a connection to the control layer. If the connection
+     * is not formed within this time, the client's network node will throw an error.
+     */
+    connectionTimeout?: number
```

Comment:
This should more clearly indicate that the configuration is only used when opening the first connection to reach the network as before.